Exec Doc 2025-09-18 (#41)
sky-ecosystem/executive-votes: 2025/executive-vote-2025-09-18.md

---
title: Template - [Executive Vote] Delayed Upgrade Penalty, Remove Nova Operator, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS Payments for September 2025, Atlas Core Development SKY Payments for September 2025, Spark Proxy Spell - September 18, 2025

> summary: Set Delayed Upgrade Penalty, Remove Nova Operator as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, Aligned Delegate Compensation, Atlas Core Development payments for September 2025, Spark proxy spell execution.

Reviewer: ```suggestion
summary: Set Delayed Upgrade Penalty, Replace Nova Operator as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, Aligned Delegate Compensation, Atlas Core Development payments for September 2025, Spark proxy spell execution.
```

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.

---

## Proposal Details

### Set Delayed Upgrade Penalty

- **Authorization**: [A.4.1.2.1.4.2.5](https://sky-atlas.powerhouse.io/A.4.1.2.1.4.2.5_Set_Conversion_Fee_In_MKR_To_SKY_Conversion_Contract_To_1%25/1f1f2ff0-8d73-804c-948b-fddc869fcb65%7Cb341f4c0b83472dc1f9e1a3b)
- **Proposal**: [Forum Post](https://forum.sky.money/t/phase-3-mkr-to-sky-migration-items-september-18th-spell/27178)

If this executive proposal passes, then the Delayed Upgrade Penalty will be set to **1%**.

> ### Remove Nova Operator

Reviewer: ```suggestion
### Replace Nova Operator
```

    - `AllocatorRolesLike(ALLOCATOR_ROLES).setRoleAction("ALLOCATOR-NOVA-A", 0, ALLOCATOR_NOVA_A_VAULT, VaultLike.wipe.selector, false);`
- Set the SubProxy address to [0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310](https://etherscan.io/address/0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310)
- Set up the Subproxy:
  - Set SubProxy as the ilk admin instead of the Pause Proxy via:
    - `AllocatorRolesLike(ALLOCATOR_ROLES).setIlkAdmin("ALLOCATOR-NOVA-A", NOVA_PROXY);`
  - Move ownership of the ALLOCATOR_NOVA_A_VAULT to the SubProxy via:
    - `WardsAbstract(ALLOCATOR_NOVA_A_VAULT).rely(NOVA_PROXY);`
    - `WardsAbstract(ALLOCATOR_NOVA_A_VAULT).deny(MCD_PAUSE_PROXY);`
  - Move ownership of the ALLOCATOR_NOVA_A_BUFFER to the SubProxy via:
    - `WardsAbstract(ALLOCATOR_NOVA_A_BUFFER).rely(NOVA_PROXY);`
    - `WardsAbstract(ALLOCATOR_NOVA_A_BUFFER).deny(MCD_PAUSE_PROXY);`

### LSEV2-SKY-A Liquidation Ratio Increase

- **Authorization**: [Atlas Axis Authorization](https://forum.sky.money/t/september-18-2025-proposed-changes-to-lsev2-sky-a-liquidation-ratio/27160/2)
- **Proposal**: [Forum Post](https://forum.sky.money/t/september-18-2025-proposed-changes-to-lsev2-sky-a-liquidation-ratio/27160)

> If this executive proposal passes, then the LSEV2-SKY-A Liquidation Ratio will be increased as follows:

Reviewer: ```suggestion
If this executive proposal passes, then the LSEV2-SKY-A [Liquidation Ratio](https://sky-atlas.powerhouse.io/A.3.8.1.1.2.1_Liquidation_Ratio/1184ab96-8cad-4755-9798-2c8782304407|57eaf45219bea3b430c2) will be increased as follows:
```

- Increase LSEV2-SKY-A Liquidation Ratio by 20 percentage points, from 125% to **145%**.

### Monthly Settlement Cycle Execution for July-August 2025

- **Authorization**: [A.2.5.1.2.2.1](https://sky-atlas.powerhouse.io/A.2.5.1.2.2.1_Stage_1/241f2ff0-8d73-8014-b124-e76f5f5c91fc%7C9e1fcc279923ea16fa2d)
- **Proposal**: [Forum Post](https://forum.sky.money/t/monthly-settlement-cycle-1-july-august-september-18-2025-spell/27173)

> If this executive proposal passes, then the Monthly Settlement Cycle corresponding to July-August 2025 will be executed as follows.

Reviewer: ```suggestion
If this executive proposal passes, then the [Monthly Settlement Cycle](https://sky-atlas.powerhouse.io/A.2.5_Sky_Core_Monthly_Settlement_Cycle/1d8f2ff0-8d73-807c-80f0-e58dde70cc27|9e1f) corresponding to July-August 2025 will be executed as follows.
```

---

> title: Template - [Executive Vote] Delayed Upgrade Penalty, Nova Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS Payments for September 2025, Atlas Core Development SKY Payments for September 2025, Spark Proxy Spell - September 18, 2025

Reviewer 2: ```suggestion
title: Template - [Executive Vote] Delayed Upgrade Penalty, Nova Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS and SKY Payments, Spark Proxy Spell - September 18, 2025
```

> title: Template - [Executive Vote] Delayed Upgrade Penalty, Nova Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS Payments for September 2025, Atlas Core Development SKY Payments for September 2025, Spark Proxy Spell - September 18, 2025

Reviewer 3: ```suggestion
title: Template - [Executive Vote] Delayed Upgrade Penalty, Launch Agent 2 Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS Payments for September 2025, Atlas Core Development SKY Payments for September 2025, Spark Proxy Spell - September 18, 2025
```

> summary: Set Delayed Upgrade Penalty, Nova Allocator Adjustment as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, August Aligned Delegate Compensation, Atlas Core Development USDS and SKY payments for September 2025, Spark proxy spell execution.

Reviewer 3: ```suggestion
summary: Set Delayed Upgrade Penalty, Launch Agent 2 Allocator Adjustment as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, August Aligned Delegate Compensation, Atlas Core Development USDS and SKY payments for September 2025, Spark proxy spell execution.
```

> summary: Set Delayed Upgrade Penalty, Launch Agent 2 Allocator Adjustment as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, August Aligned Delegate Compensation, Atlas Core Development USDS and SKY payments for September 2025, Spark proxy spell execution.

Reviewer 3: ```suggestion
summary: Set Delayed Upgrade Penalty, Launch Agent 2 Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, August Aligned Delegate Compensation, Atlas Core Development USDS and SKY payments for September 2025, Spark proxy spell execution.
```

> summary: Set Delayed Upgrade Penalty, Nova Allocator Adjustment as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, Aligned Delegate Compensation, Atlas Core Development payments for September 2025, Spark proxy spell execution.

Reviewer 2: ```suggestion
summary: Set Delayed Upgrade Penalty, Nova Allocator Adjustment as the operator of Launch Agent 2, LSEV2-SKY-A Liquidation Ratio increase, Monthly Settlement Cycle execution for July-August 2025, August Aligned Delegate Compensation, Atlas Core Development USDS and SKY payments for September 2025, Spark proxy spell execution.
```

date: 2025-09-18T00:00:00.000Z
address: "$spell_address"
---

# [Executive Proposal] Delayed Upgrade Penalty, Nova Allocator Adjustment, LSEV2-SKY-A Liquidation Ratio Increase, First Monthly Settlement Cycle, AD Compensation, Atlas Core Development USDS Payments for September 2025, Atlas Core Development SKY Payments for September 2025, Spark Proxy Spell - September 18, 2025

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. SKY holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

> - The Delayed Upgrade Penalty will be set to 1%.

Reviewer 2: ```suggestion
- The [Delayed Upgrade Penalty](https://sky-atlas.powerhouse.io/A.4.1.2.1.4.2.5_Set_Conversion_Fee_In_MKR_To_SKY_Conversion_Contract_To_1%25/1f1f2ff0-8d73-804c-948b-fddc869fcb65%7Cb341f4c0b83472dc1f9e1a3b) will be set to 1%.
```

> - Nova Operator will be removed as the operator of Launch Agent 2 in favor of a proxy-based setup.

Reviewer 3: ```suggestion
- Launch Agent 2 Operator will be removed in favor of a proxy-based setup.
```

> - Launch Agent 2 Operator will be removed in favor of a proxy-based setup.

Reviewer 3: ```suggestion
- Launch Agent 2 Allocator Instance Operator will be replaced with a subproxy-based setup.
```

This executive proposal **has its earliest execution date set to Monday September 22, 2025 at 14:00 UTC**. This means that it cannot be executed before that date.

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.

---

## Proposal Details

### Set Delayed Upgrade Penalty

- **Authorization**: [A.4.1.2.1.4.2.5](https://sky-atlas.powerhouse.io/A.4.1.2.1.4.2.5_Set_Conversion_Fee_In_MKR_To_SKY_Conversion_Contract_To_1%25/1f1f2ff0-8d73-804c-948b-fddc869fcb65%7Cb341f4c0b83472dc1f9e1a3b)
- **Proposal**: [Forum Post](https://forum.sky.money/t/phase-3-mkr-to-sky-migration-items-september-18th-spell/27178)

If this executive proposal passes, then the Delayed Upgrade Penalty for upgrading MKR to SKY will be set to **1%**.

### Launch Agent 2 Allocator Adjustment

- **Authorization**: [Governance Poll 1583](https://vote.sky.money/polling/QmYt7nbx)
- **Proposal**: [Forum Post](https://forum.sky.money/t/technical-scope-of-the-nova-allocator-adjustment/27175)

> If this executive proposal passes, then the following series of actions related to the genesis process pertaining to Launch Agent 2 will be executed:

Reviewer 3: ```suggestion
If this executive proposal passes, then the following series of actions to Launch Agent 2 will be executed:
```

If this executive proposal passes, then the LSEV2-SKY-A [Liquidation Ratio](https://sky-atlas.powerhouse.io/A.3.8.1.1.2.1_Liquidation_Ratio/1184ab96-8cad-4755-9798-2c8782304407|57eaf45219bea3b430c2) will be increased as follows:

- Increase LSEV2-SKY-A Liquidation Ratio by 20 percentage points, from 125% to **145%**.

### Monthly Settlement Cycle Execution for July-August 2025

- **Authorization**: [A.2.5.1.2.2.1](https://sky-atlas.powerhouse.io/A.2.5.1.2.2.1_Stage_1/241f2ff0-8d73-8014-b124-e76f5f5c91fc%7C9e1fcc279923ea16fa2d)
- **Proposal**: [Forum Post](https://forum.sky.money/t/monthly-settlement-cycle-1-july-august-september-18-2025-spell/27173)

If this executive proposal passes, then the [Monthly Settlement Cycle](https://sky-atlas.powerhouse.io/A.2.5_Sky_Core_Monthly_Settlement_Cycle/1d8f2ff0-8d73-807c-80f0-e58dde70cc27|9e1f) corresponding to July-August 2025 will be executed as follows.

#### Spark Settlement

- Mint 1,603,952 USDS debt in ALLOCATOR-SPARK-A and transfer the amount to the Surplus Buffer via:
  - `_takeAllocatorPayment(ALLOCATOR_SPARK_A_VAULT, 1_603_952e18);`

> - Send 5,927,944 USDS from the Surplus Buffer to the Spark SubProxy via:

Reviewer 3: ```suggestion
- Transfer 5,927,944 USDS from the Surplus Buffer to the Spark SubProxy via:
```
